app/models/finance/billing_strategy.rb

```diff
@@ -104,10 +104,6 @@ def self.daily(options = {})
         message = "BillingStrategy #{id}(#{name}) failed utterly"
 
         Rails.logger.error(message)
-        airbrake(:error_message => message,
-                 :error_class => 'BillingError',
-                 :exception => e)
-
```

Comment:
I wonder if we should do a `System::ErrorReporting.report_error` here instead. I guess not because the next line raises an error, and it is not rescued anywhere, then it will notify bugsnag (and will do a Rails logger error as well, I think).
But I don't know if it is rescued somewhere or not. Maybe @guicassolato knows.

Reply:
In other places the error was reported manually.. so it would make sense to me to do it here too.

Reply:
No need if it raises the error unless you want to add cuatom data to the error